package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "asab_webui_components",
-	"version": "25.2.4",
+	"version": "25.3.1",
```

Comment:
@aringocode Ithink this is more like `25.2.6` version, since this MR does not introduce any new component feature or significant refactorisation. Also please merge main branch to yours, seems like you dont have the latest changes